### PR TITLE
Use correct post dates

### PR DESCRIPTION
**Pull request by annalinneajohansson**

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-3qnp7u7 fel publiceringsdatum posts-modul kulturkortet.se startsida](https://app.clickup.com/t/3qnp7u7)

File: source/php/Module/Posts/views/features-grid.blade.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
                 'content' => ($post->showExcerpt ? $post->post_content : false),
                 'link' => $post->link,
                 'meta' => $post->tags,
-                'date' => ($post->showDate ? $post->post_date : false),
+                'date' => $post->showDate ? date("Y-m-d H:i", strtotime($post->post_date)) : false,
```
Comment (sebastianthulin) [inline source/php/Module/Posts/views/features-grid.blade.php:11]: Use WordPress date format set in settings panel.

File: source/php/Module/Posts/views/partials/date.blade.php [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@if($post->postDate)
+@if($post->postDate)z
```
Comment (sebastianthulin) [inline source/php/Module/Posts/views/partials/date.blade.php:1]: z?

**Review by sebastianthulin — CHANGES_REQUESTED** [review]
Needs to use WordPress date format from settings.

**Pull request merged**